vocabularies/PersonalData.xml

```diff
@@ -179,6 +179,15 @@ Terms for contact and address information are defined in the [Communication voca
                 <String>A user is an individual who interacts with the services supplied by a system.</String>
               </Annotation>
             </Record>
+            <Record>
+              <Annotation Term="Common.Experimental" />
+              <PropertyValue Property="Value" String="EndOfBusinessDate" />
```

Comment:
"end of business date" sounds rather generic: what is the official GDPR terminology?

Reply:
"End of purpose" perhaps? That's the date after which you no longer need the data and must therefore delete it.